language_service/src/compilation.rs

```diff
@@ -312,6 +278,25 @@ fn run_fir_passes(
     }
 }
 
+/// Compute new lints and append them to the errors Vec.
+/// Lints are only computed if the erros vector is empty. For performance
+/// reasons we don't want to waste time running lints every few keystrokes,
+/// if the user is in the middle of typing a statement, for example.
+fn run_linter_passes(
+    config: Option<&[LintConfig]>,
```

Comment:
@orpuente-MS  We are only ever passing `Some(..)` into this call. Can we drop the option type here?

Reply:
Good catch, already fixed. Thanks!